src/components/users/signup.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable */
+
+import React, { Component } from 'react';
+import axios from 'axios';
+class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      name: '',
+      email: '',
+      password: '',
+      password_confirmation: '',
+      errors: ''
+     };
+  }
+handleChange = (event) => {
```

Comment:
- [ ] This function should be indented, since it's inside of the `Signup` component, and this is happening in other places as well, I suggest that you check all of them and make sure they are properly indented.

Reply:
-[ ] Hello dear @MuriloRoque , Thank you for your reviews, some of your reviews were resourceful for us. but, It seems that you didn't check very well some project requirements and some of our components. We used `Class component` just for authentication the user means in the `users ` directory. This part was not required for us because we're just a team of 3 people but we implemented it to make our project looks more professional. 

- [ ] Also, I think that the test part should be **[ OPTIONAL ]** for this part because is not mandatory in the project requirement. We implemented some tests in the backend part to be sure that our API is reliable and can be used by other people without more complications.

Best,